### PR TITLE
Sigxill feed tags new

### PR DESCRIPTION
**Pull request by orhovy**
## Status
- [x] Ready

## Related Issues
https://github.com/demisto/content/pull/8307

## Description
Added tags param to feed.

## Minimum version of Demisto
- [x] 5.5.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests

### CODE REVIEW

File: Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed/Sixgill_Darkfeed_test.py [hunk 1]
```diff
@@ -531,3 +523,32 @@ def test_get_indicators_command(mocker):
...
+    from sixgill.sixgill_constants import FeedStream
+
+    client = SixgillFeedClient("client_id",
+                               "client_secret",
+                               "some_channel",
+                               FeedStream.DARKFEED,
+                               demisto, 1000)
+
+    output = fetch_indicators_command(client, tags=['tag1', 'tag2'])
+    assert all(item in output[0]['fields']['tags'] for item in ['tag1', 'tag2'])
```
Comment (bakatzir) [inline Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed/Sixgill_Darkfeed_test.py:554]: does some of the indicators contain yet another tag? e.g: compromised,malware hash, shell
please ass a test case taht validates that the `union` works as expected and both `feedTags` and the IOC `labels` are present. in an indicator of your choice.

**Comment by bakatzir** [comment]
@anara123 FYI as you reviewed the original feed.

File: Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed/Sixgill_Darkfeed_test.py (hunk 1 above)
Comment (orhovy) [reply Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed/Sixgill_Darkfeed_test.py:554]: Yes they all contain a tag and the test checks that the 'tag1' and 'tag2' are added

File: Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed/Sixgill_Darkfeed_test.py [hunk 2]
```diff
@@ -531,3 +523,32 @@ def test_get_indicators_command(mocker):
     submitted_indicators = 0
 
     assert output[2] == expected_ioc_output
+
+
+def test_feed_tags(mocker):
+    """
+    Given:
+    - tags parameters
```
Comment (bakatzir) [inline Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed/Sixgill_Darkfeed_test.py:531]:
```suggestion
    - feedTags parameter
```

File: Packs/Sixgill-Darkfeed/pack_metadata.json [hunk 3]
```diff
@@ -1,16 +1,16 @@
 {
-    "name": "Darkfeed™ - Current Customer Edition",
+    "name": "Darkfeed\u2122 - Current Customer Edition",
```
Comment (bakatzir) [inline Packs/Sixgill-Darkfeed/pack_metadata.json:2]: please verify on the 6.0.0 build that this pack name looks ok.

File: Packs/Sixgill-Darkfeed/pack_metadata.json [hunk 4]
```diff
@@ -1,16 +1,16 @@
 {
-    "name": "Darkfeed™ - Current Customer Edition",
+    "name": "Darkfeed - Current Customer Edition",
```
Comment (bakatzir) [inline Packs/Sixgill-Darkfeed/pack_metadata.json:2]: @yuvalbenshalom / @amshamah419 WDYT about the `tm`? is it ok to change it?

**Pull request merged**